README.md

```diff
@@ -2,6 +2,19 @@
 
 Script to create a certificate chain with self-signed CA where the client certificate is compliant with the [standard](https://www.etsi.org/deliver/etsi_en/319400_319499/31941203/01.02.01_60/en_31941203v010201p.pdf).
 
+## How to use
+
+Run the container and mount local output folder
+```
+   docker run --mount type=bind,source="$(pwd)"/output,target=/out quay.io/pulledtim/eidas:latest 
```

Comment:
Just a small remark, the 'output' folder should be created manually before, otherwise the command produces error. So would be probably better to add this hint or adjust the command so that it automatically creates a folder if it's missing